content/html/concepts/tags/terms/kbd/kbd.md

```diff
@@ -0,0 +1,37 @@
+---
+Title: '<kbd>'
+Description: 'Makes charecters look like keys on a keyboard'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Tags'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `<kbd>` tag makes characters look like they are keys on a keyboard.
+
+## Syntax
+
+All that is needed to be done is put the characters between an opening and closing tag. The `<kbd>` tag has no attributes. They can be nested but the result will be the same.
+
+## Example 1
+
+```html
+<p>You can copy text by pressing <kbd>CTRL</kbd> + <kbd>c</kbd>.</p>
+```
+
+Result:
+
+You can copy text by pressing ``CTRL`` + ``c``.
```

Comment:
This is a case where we could use an image of the rendered text. The picture can be placed in the /media folder.